composer.json

```diff
@@ -13,16 +13,16 @@
     "crossbar"
   ],
   "require": {
-    "php": "^7.0|^8.0",
-    "symfony/http-kernel": "^2.7 || ^3.0 || ^4.0",
-    "symfony/framework-bundle": "^2.7 || ^3.0 || ^4.0",
-    "symfony/dependency-injection": "^2.7 || ^3.0 || ^4.0",
-    "symfony/config": "^2.7 || ^3.0 || ^4.0",
+    "php": "^7.4 || ^8.1",
```

Comment:
Why is PHP 8.0 dropped? I know it's EOL, but if we're still supporting 7.4, it doesn't make sense to drop only that one.

Reply:
I would see this with @ilario-pierbattista